src/routes/Router.tsx

```diff
@@ -1,14 +1,13 @@
-import React from 'react';
-import { Route, Routes } from 'react-router-dom';
-import PageTemplate from '../pages/PageTemplate/PageTemplate.tsx';
+import React from "react";
+import { Route, Routes } from "react-router-dom";
 
-function Router() {
+export const Router: React.FC = () => {
   return (
-    <Routes>
-      <Route path="/" element={<PageTemplate />} />
-      {/* Add all Routes here */}
-    </Routes>
+    <>
+      <Routes>
+        <Route path="/" />
+        <Route path="/phones" element={/* Phone Comp */} />
```

Comment:
This code ... element={/* Phone comp */} ... leads to an error.

Element prop expected a ReactNode object type. 

A non execution error way: 
....
{/* Put coments here */}
<Route path="/phones" /> {/*Coments here */}
...